coffeeRunner/coffeerun-starter/scripts/main.js

```diff
@@ -8,13 +8,40 @@ var repopulateButton = document.querySelector('button.btn.btn-default.right');
 
 
 
-$.get(url, function(data) { 
+// $.get(url, function(data) { 
+//     orders = data;
+//     for (var key in orders) {
+//         displayOrder(orders[key]);
+//     }
+// });
+
+// var getOrders = function() {
+//     var promise = new Promise(function(resolve) {
+//         $.get(url, resolve);
+//     })
+//     return promise;
+// }
+
+var getOrders = new Promise( function(resolve) {
```

Comment:
Looks good to me. Seems very clean and makes complete sense for your implementation 👏🏻

Reply:
yes, I could! good catch, I didn't think of that

Reply:
thanks!